addons/mrp/models/mrp_workorder.py

```diff
@@ -25,6 +25,8 @@ class MrpWorkorder(models.Model):
     working_state = fields.Selection(
         'Workcenter Status', related='workcenter_id.working_state', readonly=False,
         help='Technical: used in views only')
+    production_id = fields.Many2one('mrp.production', 'Manufacturing Order',
```

Comment:
I would rather add an unlink on mrp.production because this would erase workorders done.